src/tests/dictionaryConcepts/container/EditConcept.test.jsx

```diff
@@ -589,4 +589,39 @@ describe('Test suite for mappings on existing concepts', () => {
     wrapper.find('button#removeAnswer').simulate('click');
     expect(props.removeAnswer).toHaveBeenCalledWith(props.selectedAnswers[0].frontEndUniqueKey);
   });
+  it('should create a new mapping with `SAME-AS` as the default relationship when `CIEL`'
+  + 'is the default source while creating a new concept', () => {
+    const event = {
+      target: {
+        tabIndex: 0,
+        name: 'source',
+        value: 'a234',
+      },
+    };
+    const url = '1234';
+    const instance = wrapper.find('EditConcept').instance();
+    instance.state.mappings[1] = {
+      source: INTERNAL_MAPPING_DEFAULT_SOURCE,
+    };
+    instance.updateSourceEventListener(event, url);
+    expect(instance.state.mappings[1].map_type).toEqual('SAME-AS');
+  });
+  it('should edit an existing mapping with `NARROWER-THAN` as the default relationship when the source '
+  + 'is changed from `CIEL` to `Datatypes` the default source while editing a new concept', () => {
+    const event = {
+      target: {
+        tabIndex: 0,
+        name: 'source',
+        value: 'a234',
+      },
+    };
+    const url = '1234';
+    const instance = wrapper.find('EditConcept').instance();
+    instance.state.mappings[1] = {
+      source: INTERNAL_MAPPING_DEFAULT_SOURCE,
+    };
+    instance.updateSourceEventListener({ target: { tabIndex: 0, name: 'source', value: 'Datatypes' } });
+    instance.updateSourceEventListener(event, url);
+    expect(instance.state.mappings[1].map_type).toEqual('NARROWER-THAN');
```

Comment:
What value did this have before changing to NARROWER-THAN?

Reply:
I have adjusted the tests to better reflect this